cypress/page/login.page.ts

```diff
@@ -0,0 +1,24 @@
+class LoginPage {
+    private txtEmailAddress : string;
+    private txtPassword : string;
+    private btnSubmitLogin : string;
+
+    constructor() {
+        this.txtEmailAddress ="#email";
+        this.txtPassword = "#passwd";
+        this.btnSubmitLogin ="#SubmitLogin > span"
+    }
+
+    public goToEmailAddress() : void{
+        cy.get(this.txtEmailAddress).type("aperdomobo@gmail.com");
+    }
+
+    public goToPassword() : void{
+        cy.get(this.txtPassword).type("WorkshopProtractor");
```

Comment:
You should create a single method for this, in that matter you receive the variables as parameters and you define them in the spec file